src/main/java/de/tum/in/www1/artemis/repository/ExerciseGroupRepository.java

```diff
@@ -23,7 +23,11 @@
 public interface ExerciseGroupRepository extends JpaRepository<ExerciseGroup, Long> {
 
     @EntityGraph(type = LOAD, attributePaths = { "exercises" })
-    @Query("SELECT e FROM ExerciseGroup e WHERE e.id = :exerciseGroupId")
+    @Query("""
+            SELECT e
+            FROM ExerciseGroup e
+            WHERE e.id = :exerciseGroupId
+            """)
     Optional<ExerciseGroup> findWithExercisesById(@Param("exerciseGroupId") Long exerciseGroupId);
```

Comment:
Are you sure we even need the query here? This seems simple enough that spring can derive this.